web/src/blobs/blobs.module.css

```diff
@@ -168,3 +172,28 @@
 .imageUploadGroup {
   margin-top: 3rem;
 }
+
+.textField {
+  width: 100%;
+  font-weight: normal;
+  color: var(--color-standard__white--text);
+  background-color: var(--color-standard__white);
+  /* transition: all 200ms ease-in; */
+
+  border: 1px solid var(--color-secondary4__shade2);
+  border-radius: 0.2rem;
+  padding: 0.1rem 0.4rem;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  padding-right: 20px;
+}
+
+.textField:focus,
+.textField:hover {
+  border-color: var(--color-secondary4__shade1);
+}
+
+.buttonGroup {
+  display: flex;
```

Comment:
This makes the button group for download/copy in the footer not work properly:

<img width="600" alt="Screenshot 2020-06-02 at 21 03 16" src="https://user-images.githubusercontent.com/606374/83559220-7dfec900-a514-11ea-9fc1-4f7d0e482187.png">

Should probably have a more specific name.

Reply:
Well, it shouldn't have been there in the first place :sweat_smile: relic from a test to have the buttons inline with the input field (Too cramped).